commands/system/up.sh

```diff
@@ -36,6 +31,16 @@ function system:up() {
 
     ${DOCKER_COMPOSE} up -d
 
+    cd services/conf.d
+
+    for f in *; do
+        if [ -f ${ROOT_DIR}/services/${f}/docker-compose.yml ]; then
+            _logGreen "Starting service ${f}"
+            cd ${ROOT_DIR}/services/${f}
```

Comment:
This can be handled better

Reply:
It should be resolved by the commit  [879864e](https://github.com/whatwedo/dde/pull/38/commits/879864e5b5dc2fc237646da5ba243615f14293b5)